virtwho.py

```diff
@@ -175,50 +240,67 @@ def run(self):
             return
 
         result = {}
+        report = None
         while not self.terminate_event.is_set():
             # Wait for incoming report from virt backend
             try:
-                report = self.queue.get(block=True, timeout=None)
+                report = self.queue.get_nowait()
```

Comment:
I agree that for the case that it is currently used having timeouts for each job seems a little excessive. However, it will likely be very useful to have later when we implement rate-limiting.